crates/next-api/src/app.rs

```diff
@@ -732,7 +732,7 @@ pub fn app_entry_point_to_route(
 }
 
 #[turbo_tasks::function]
-fn client_shared_chunks() -> Vc<RcStr> {
+fn client_shared_chunks_mod() -> Vc<RcStr> {
```

Comment:
the shortcut isn't that helpful for readablility

```suggestion
fn client_shared_chunks_modifier() -> Vc<RcStr> {
```